tests/docs-screenshots.spec.ts

```diff
@@ -136,7 +137,7 @@ test("networks", async ({ page }) => {
     clip: getClipPosition(240, 0, 1420, 750),
   });
   await page.getByRole("button", { name: "Create", exact: true }).click();
-  await page.waitForSelector(`text=Network ${network} created.`);
+  await page.waitForTimeout(2500);
```

Comment:
Maybe instead of duplicating we can define a `submitCreateNetwork` helper that extracts those lines from the `createNetwork` helper, so we can reuse it here and avoid the explicit timeout in the test:

```
  await page.waitForTimeout(750);
  await page.getByRole("button", { name: "Create", exact: true }).click();
  await page.waitForTimeout(2500);
  const networkLink = await getNetworkLink(page, network);
  await expect(networkLink).toBeVisible();
  ```